workers/unity/Assets/Playground/Scripts/Player/PlayerCommandsSystem.cs

```diff
@@ -86,8 +106,12 @@ protected override void OnUpdate()
                         command == PlayerCommand.LaunchLarge ? LargeEnergy : SmallEnergy,
                         playerId
                     ));
-
+#if UNITY_EDITOR
+                lastId = commandSystem.SendCommand(request, entities[0]);
+                Debug.Log($"Launching {lastId.Value.Raw}");
+#else
```

Comment:
Just checking if you tested this works? I _think_ this now means that it won't send commands in the editor

Reply:
😕 if editor send command, store id and log else send command